examples/with-nextjs-and-clerk-auth/src/themes/monite.ts

```diff
@@ -7,6 +7,8 @@ import type {
 import type { Theme } from '@mui/material/styles/createTheme';
 import type { TypographyOptions } from '@mui/material/styles/createTypography';
 import { deepmerge } from '@mui/utils';
+import '@mui/x-data-grid/themeAugmentation';
+import { DataGridComponentNameToClassKey } from '@mui/x-data-grid/themeAugmentation/overrides';
```

Comment:
`DataGridComponentNameToClassKey` could be removed.
`themeAugmentation` is handling it.
I tested commands `build`, `typecheck`. Everything works without `DataGridComponentNameToClassKey`